test/Conversion/TorchToLinalg/onehot.mlir

```diff
@@ -0,0 +1,42 @@
+// RUN: torch-mlir-opt --pass-pipeline="builtin.module(func.func(torch-decompose-complex-ops),torch-backend-to-linalg-on-tensors-backend-pipeline)" -verify-diagnostics < %s | FileCheck %s
+
+// -----
```

Comment:
You don't need this separator if you only have one test module. This separator is to allow for multiple modules in a single test, and this is enabled in the compiler using the `--split-input-file` flag.